packages/minuteman/extra/dcos-minuteman.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+WorkingDirectory=/opt/mesosphere/active/minuteman/minuteman
+EnvironmentFile=/opt/mesosphere/etc/minuteman-erl.env
+EnvironmentFile=/opt/mesosphere/etc/check_time.env
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/minuteman.env
+EnvironmentFile=-/run/dcos/etc/minuteman_auth.env
+ExecStartPre=/opt/mesosphere/bin/check-time
+ExecStartPre=/usr/bin/env modprobe ip_vs_wlc
+ExecStartPre=/opt/mesosphere/bin/setup_iptables.sh
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-minuteman
+ExecStartPre=/bin/ping -c1 ready.spartan
+ExecStartPre=/bin/ping -c1 leader.mesos
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/minuteman/mnesia
```

Comment:
This would all end up getting owned by root because the ExecStartPre is run as root, which isn't what we want. Minuteman process probably needs to do the mkdir as part of it's startup.